app/src/main/java/org/simple/clinic/summary/PatientSummaryConfig.kt

```diff
@@ -5,11 +5,13 @@ import org.threeten.bp.Duration
 import javax.inject.Inject
 
 data class PatientSummaryConfig(
-    val bpEditableDuration: Duration
+    val bpEditableDuration: Duration,
+    val numberOfMeasurementsForDoctor: Int
```

Comment:
I used measurements so that it can be open to use for blood sugars in future if needed 🤔 . Should I change it `numberOfBPsForTeleconsultation` for now?

Reply:
We don't know if we might need a separate field for blood sugars since the numbers could be different for BPs and blood sugars. 

I want to get rid of the word `Doctor` since this does not tell me that what are we fetching the measurements for? Since doctors are also our users, this may not make a lot of sense in that context.

Reply:
Yeah, I missed that part 😅 . I pushed the changes